libs/output-mapping/src/Writer/Helper/SliceCommandBuilder.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\OutputMapping\Writer\Helper;
+
+use SplFileInfo;
+
+class SliceCommandBuilder
+{
+    public static function create(
```

Comment:
od SliceCommandBuilder::create bych asi očekával, že vrátí Command a ne array

Reply:
No argument u Process je command a je to array :) Udelal jsem to teda at to zaklada rovnou ten process, ale moc se mi to nelibi.

Reply:
mě to takhle přijde lepší - navíc nemusíš zvenku myslet na to, že je potřeba tam nastavit ten timeout